### PR TITLE
Final version of dense net

### PR DESCRIPTION
**Pull request by e33624303**
1. Modify some part of  the model architecture
2. Plot the training loss and accuracy
3. Add learning rate scheduler

### CODE REVIEW

**Comment by laituan245** [comment]
Thank you very much for your hard work.
